File: Backend/TodoApp.API/TodoApp.API/TodoApp.API/Models/Todo.cs
using System.ComponentModel;

namespace TodoApp.API.Models
{
    public class Todo
    {
        public Guid id { get; set; }
        public Guid ListID { get; set; }
        public string name { get; set; }
        [DefaultValue(false)]
        public bool isDone { get; set; }
        public int priority { get; set; }
        public List List { get; set; }
        [DefaultValue(false)]
        public bool isDeleted { get; set; }
        
    }
}

File: FrontEnd/ToDoListProject/src/app/models/todo.ts
export class Todo {
  id?: string;
  ListID!: string;
  name!: string;
  isDone: boolean = false;
  priority!: number;
  isDeleted: boolean = false;

  constructor(init?: Partial<Todo>) {
    Object.assign(this, init);
  }
}
